source/common/router/router.h

```diff
@@ -371,19 +369,25 @@ class Filter : Logger::Loggable<Logger::Id::router>,
                                          Upstream::ResourcePriority priority) PURE;
   Http::ConnectionPool::Instance* getConnPool();
   void maybeDoShadowing();
+  bool maybeRetryReset(Http::StreamResetReason reset_reason);
+  void onPerTryTimeout();
   void onRequestComplete();
   void onResponseTimeout();
   void onUpstream100ContinueHeaders(Http::HeaderMapPtr&& headers);
+  // Handle an "aborted" upstream request, meaning we didn't see response
+  // headers (e.g. due to a reset). Handles recording stats and responding
+  // downstream if appropriate.
+  void onUpstreamAbort(Http::Code code, StreamInfo::ResponseFlag response_flag, absl::string_view body, bool dropped);
```

Comment:
Something funky is happening here, this is line is too long and shouldn't pass the formatter. Are we missing a clang-format enable somewhere above? Can you take a look?

Reply:
hmm I don't see any mentions to clang-format in this file, and enabling it doesn't seem to make a difference. I'll wrap it manually

Reply:
nvm I was looking in the wrong file, but the last clang-format directive is on